examples/DotNetCore/WebDemo/Microsoft.Azure.AppConfiguration.WebDemo/appsettings.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "Logging": {
+    "IncludeScopes": false,
+    "LogLevel": {
+      "Default": "Warning"
+    }
+  },
+
+  "Settings": {
+    "AppName": "Azure App Configuration Demo",
+    "Version": 1.0,
+    "FontSize": 50,
+    "RefreshRate": 1000,
+    "Language": "English",
+    "Messages": "Hello There;Thanks For Using Azure App Configuration",
+    "BackgroundColor": "Orange"
+  },
```

Comment:
Please remove this, the commented section was added to replace this.

Reply:
Removed.